book/website/reproducible-research/open/open-hardware.md

```diff
@@ -1,111 +1,359 @@
+
+    - **Manufacturing instructions**: The manufacturing process of parts that have been made for this project should be documented as well. This is specially important if they are available for purchase from only a handful of small/medium businesses.
+
+- A **list of required tools** and associated settings, for both software used for development, and machine tools for replication.
+
+- **Design files** with parts metadata, typically including the manufacturing process, the materials with dimensions, mass, and units. A functional overview of the project's parts/modules can also be included. Ideally, your OSH project would be designed using a free and open source software application, to maximize the ability of others to view and edit it. It is essential to share these original design files, in both original and accessible ready-to-view formats. The type of parts can also be mentionned with unambiguous reference, between, custom parts (e.g. developed as a result of this or another project), off-the-shelf parts (e.g. screws) or (maybe proprietary) complex modules (for example, a single board computer). 
+
+-  **Software and firmware**. You should share any code or firmware required to operate your hardware. This will allow others to use it with their hardware or modify it along with their modifications to your hardware. Document the process required to build your software, including links to any dependencies (for example, third-party libraries or tools). In addition, it is helpful to provide an overview of the state of the software (for example, “stable” or “beta” or “barely-working hack”). 
+
+- **Instructions for operation and maintenance**. Describe how hardware users should operate the hardware (for example how to calibrate and test it). Indicate any maintenance that should be done to secure a good functionality of the hardware.
+
+- **Repair and disposal**. Indicate where or how the hardware can be repaired, and indicate how to dispose or recycle the hardware, if it is beyond repair.
+
+> Note that producing documentation-quality pictures consistently requires adequate tools and setup.
+
+
+## Community documentation
+
+
+While open hardware communities are sometimes different than open oftware communities, the documentation useful to grow hardware communities are similar as the one for other open source project, see the  {ref}`pd-overview-repro` chapter. You may want to refer to the {ref}`collaboration<cl>`  and {ref}`cl-new-community`  guide of the turing way book for a more detailed description of certain aspects such as practices, metrics, behaviors and observables that can be related to thriving communities.
+
+OSH communities comes in all sizes and forms.
+They often develop around people facing similar needs, who realise they will get "something that none of [them] could have developed alone." (Interview: White rabbit, by the Open make team, Javier Serrano and Amanda Diez Frenandez.)
```

Comment:
Are you meaning to link or cite this interview references in parentheses?

Reply:
```suggestion
They often develop around people facing similar needs, who realise they will get "something that none of [them] could have developed alone." ([Interview: White rabbit, by the Open make team, Javier Serrano and Amanda Diez Frenandez](https://www.openmake.de/blog/2022/10/20/2022-10-06-interview-white-rabbit/) .)
```